### PR TITLE
AdminID

### PR DESCRIPTION
**Pull request by pheobeayo**
## Description

  Added the admin section for copying

## Related Issue

 https://github.com/bandada-infra/bandada/issues/588

## Does this introduce a breaking change?

-   [ ] Yes
-   [ No] No

### CODE REVIEW

File: apps/dashboard/src/pages/group.tsx [hunk 1]
```diff
@@ -52,11 +52,15 @@ export default function GroupPage(): JSX.Element {
...
     const { groupId, groupType } = useParams()
+    const {adminId, adminType } = useParams()
     const [_group, setGroup] = useState<Group | null>()
     // const { hasCopied, setValue: setApiKey, onCopy } = useClipboard("")
     const { hasCopied: hasCopiedGroupId, onCopy: onCopyGroupId } = useClipboard(
         groupId || ""
     )
+    const { hasCopied: hasCopiedAdminId, onCopy: onCopyAdminId } = useClipboard(
+        adminId || ""
+    )
```
Comment (vplasencia) [inline apps/dashboard/src/pages/group.tsx:63]:
```suggestion
       const { hasCopied: hasCopiedAdminId, onCopy: onCopyAdminId } = useClipboard(
        (_group && _group.admin) || ""
    )
```

I think we can remove the following line: `const {adminId, adminType } = useParams()` and update this `useClipboard` hook and add it after `const isGroupAdmin`.

**Review by vplasencia — CHANGES_REQUESTED** [review]
Hey @pheobeayo looks great! Thank you very much for creating this PR. I just left one comment.

File: apps/dashboard/src/pages/group.tsx (hunk 1 above)
Comment (pheobeayo) [reply apps/dashboard/src/pages/group.tsx]: done @vplasencia

File: apps/dashboard/src/pages/group.tsx [hunk 2]
```diff
@@ -487,6 +491,46 @@ ${memberIds.join("\n")}
...
+                        p="25px 30px 25px 30px"
+                        borderRadius="8px"
+                    >
+                        <Text fontSize="20px">Admin ID</Text>
+
+                        <InputGroup size="lg" mt="10px">
+                            <Input
+                                pr="50px"
+                                placeholder="Admin ID"
+                                value={adminId}
```
Comment (vplasencia) [inline apps/dashboard/src/pages/group.tsx]:
```suggestion
                                value={_group.admin}
```

**Review by vplasencia — CHANGES_REQUESTED** [review]
Could you also run `yarn prettier:write` and push the changes?

File: apps/dashboard/src/pages/group.tsx [hunk 3]
```diff
@@ -52,11 +52,15 @@ export default function GroupPage(): JSX.Element {
     const addMembersModal = useDisclosure()
     const toast = useToast()
     const { groupId, groupType } = useParams()
+
```
[the remark this answers is not in the document]
Comment (pheobeayo) [reply apps/dashboard/src/pages/group.tsx:55]: okay

**Comment by pheobeayo** [comment]
changes maded

**Review by vplasencia — APPROVED** [review]
Hey @pheobeayo great work! Thank you very much!

**Comment by vplasencia** [comment]
Hey @pheobeayo the PR is correct, but there is an issue in the CI with the style, could you run `yarn prettier:write` and push the changes to fix it?

**Comment by pheobeayo** [comment]
okay

**Pull request merged**